react/LocaleSwitcher.js

```diff
@@ -1,3 +1,3 @@
-import LocaleSwitcher from './components/index'
+import LocaleSwitcher from './components'
```

Comment:
Why not just move the whole implementation to the root and delete `/components`, since it's only one component?

Reply:
If I do that, I would have to pass all the code to the file that is in the root, doesn't seem right

Reply:
What's the problem with doing that? 
It would function exactly the same, would just remove the need for this import.
Not even the `interfaces` file would change, you would still be exporting the exact same code, but with one less "hop" to actually find the code